src/main/kotlin/org/rust/lang/core/psi/RsPsiFactory.kt

```diff
@@ -423,12 +424,14 @@ class RsPsiFactory(
             ?: error("Failed to create pat element")
 
     fun createPatField(name: String): RsPatField =
-        createFromText("""
+        createFromText(
+            """
             struct Foo { bar: i32 }
             fn baz(foo: Foo) {
                 let Foo { $name } = foo;
             }
-        """) ?: error("Failed to create pat field")
+        """
+        ) ?: error("Failed to create pat field")
```

Comment:
it looks like an unintentional change

Reply:
Kinda, but this is what the Idea autoformatting does. It is tedious to wrestle with it, so I suggest either changing the formatting rules, if possible, or surrendering to it.